app/assets/stylesheets/15-song-card-dropdown.scss

```diff
@@ -1,5 +1,10 @@
 .song-card-dropdown {
-    left: 143px;
-    top: 280px;
-    align-items: flex-start;
+    // left: 143px;
```

Comment:
nit: did you mean to leave these commented

Reply:
Yes, this is just an initial styling so something blatant renders. Was just checking state changes between parent-child.